src/base/AsyncJob.cc

```diff
@@ -199,15 +197,14 @@ AsyncJob::ReportAllJobs(StoreEntry *e)
 {
     PackableStream os(*e);
     // this loop uses YAML syntax, but AsyncJob::status() still needs to be adjusted to use YAML
-    const char *indent = "    ";
+    const std::string indent("  ");
     for (const auto job: AllJobs()) {
-        os << indent << job->id << ":\n";
-        os << indent << indent << "type: '" << job->typeName << "'\n";
-        os << indent << indent << "status:" << job->status() << '\n';
-        if (!job->started_)
-            os << indent << indent << "started: false\n";
-        if (job->stopReason)
-            os << indent << indent << "stopped: '" << job->stopReason << "'\n";
+        os << job->id << ":\n";
+        os << indent << "type: " << job->typeName << '\n';
```

Comment:
You may want to review discussions that led to this recently added code. Did we make a mistake when deciding that typeName should be quoted?

Reply:
added back